e2etests/tests/advertise_and_receive.go

```diff
@@ -77,7 +77,7 @@ var _ = ginkgo.Describe("Advertising and Receiving routes", func() {
 			config := frrk8sv1beta1.FRRConfiguration{
 				ObjectMeta: metav1.ObjectMeta{
 					Name:      "test",
-					Namespace: "default",
+					Namespace: k8s.FRRK8sNamespace,
```

Comment:
this depends on the outcome of the discussion for the namespace

Reply:
removed the namespace validation, but keeping this because of https://github.com/metallb/frr-k8s/pull/31#discussion_r1286140760 (changed the commit message)